variables.mk

```diff
@@ -75,6 +75,7 @@ LDFLAGS := -ldflags "-X ${METRICS_PACKAGE}.Revision=${RELEASE_STR} -X ${METRICS_
 KIND_DIR := test/kind
 KIND     := ${KIND_DIR}/kind
 KIND_VER := 0.8.1
+KIND_NODE_VER := v1.19.11
```

Comment:
Is it possible to avoid duplication here - https://github.com/dell/csi-baremetal/blob/master/.github/workflows/e2e.yml#L16?